From 5e33d99c3948cd0e42a0fcad358abc68f524075b Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Wed, 31 May 2017 22:25:37 -0500
Subject: [PATCH 1824/5242] MLK-14995: ARM: dts: nand-on-flash flag set in
 wrong device node

commit  1bcb6e32c520e287b124a154207c17de4cf9ded5 from
https://source.codeaurora.org/external/imx/linux-imx.git

nand-on-flash-bbt flag for i.mx6sx sabreauto dtb was set in wrong device
node, move it back to gpmi node.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 70c255b..9e58f05 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -318,6 +318,7 @@
 &gpmi {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_gpmi_nand_1>;
+	nand-on-flash-bbt;
 	status = "okay";
 };
 
@@ -1183,7 +1184,6 @@
 	dcic_id = <1>;
 	dcic_mux = "dcic-lvds";
 	status = "okay";
-	nand-on-flash-bbt;
 };
 
 &vadc {
-- 
1.7.9.5

